stdlib/number.gr

```diff
@@ -305,3 +306,20 @@ export let rec isInfinite = (x: Number) => {
   Memory.decRef(WasmI32.fromGrain(isInfinite))
   ret
 }
+
+/**
+ * Parses a string representation of an integer into a `Number` using the
+ * specified radix (also known as a number system "base").
+ * 
+ * If the string has a radix prefix (i.e. "0x"/"0X", "0o"/"0O", or "0b"/"0B"
+ * for radixes 16, 8, or 2 respectively), the supplied radix is ignored in
+ * favor of the prefix. Underscores that appear in the numeric portion of the
+ * input are ignored.
+ * 
+ * @param input: The string to parse
+ * @param radix: The number system base to use when parsing the input string
+ * @returns `Ok(value)` on a successful parse or `Err(msg)` on an unsuccessful parse, with `value` as the parsed number or `msg` as an error message string.
```

Comment:
```suggestion
 * @returns `Ok(value)` containing the parsed number on a successful parse or `Err(msg)` containing an error message string otherwise
```

I believe this is the convention